lib/insights/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -11,7 +11,7 @@ def self.included(other)
               errors = Insights::API::Common::ErrorDocument.new.tap do |error_document|
                 exception_list_from(exception).each do |exc|
                   code = exc.respond_to?(:code) ? exc.code : error_code_from_class(exc)
-                  error_document.add(code, "#{exc.class}: #{exc.message}")
+                  error_document.add(code.to_s, "#{exc.class}: #{exc.message}")
```

Comment:
The only real issue with this change: our kibana indexes will need to be rebuilt, most likely for all services using this gem.

That can be done, but it should be planned pretty quickly after this change is pulled into a microservice. Talking to @coderbydesign it is tough to troubleshoot why Kibana isn't returning what you expect with existing filters.

I'm also thinking filters would need to adjusted to handle the string based status code, but that is an assumption.

Reply:
Prometheus rules may be fine with this change.  It seemed that the gateway normalizes status codes.

Reply:
@syncrou Not sure how this would cause problems in Kibana it is just a message string, so if the status started as a string or an integer it should not matter. Its just about conformance to the IPP in the payload that is returned. The JSON-API uses status as string.